osu.Game/Graphics/UserInterface/OsuModSettingsTextBox.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Graphics.Colour;
+using osu.Framework.Input.Events;
+
+namespace osu.Game.Graphics.UserInterface
+{
+    public class OsuModSettingsTextBox : OsuTextBox
```

Comment:
I would just put this as a nested class inside `SettingsTextBox`, doesn't make sense to be exposed in such form and incorrectly scoped to "Mods" as mentioned elsewhere.